.github/workflows/ci.yaml

```diff
@@ -144,6 +144,23 @@ jobs:
         run: |
           google-chrome --version
 
+      - name: Add LLVM repository (Ubuntu 20.04 only)
+        if: matrix.os == 'ubuntu-20.04'
+        run: |
+          wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+          sudo add-apt-repository "deb http://apt.llvm.org/focal/ llvm-toolchain-focal-14 main"
+          sudo apt-get update
+
+      - name: Install LLVM and Clang
```

Comment:
nit: I propose to say "Install llvm-symbolizer" to make the why clearer.

Reply:
Done!